shared/actions/chat/attachment.js

```diff
@@ -25,7 +25,10 @@ function * onShareAttachment ({payload: {message}}: Constants.ShareAttachment):
 function * onSaveAttachmentNative ({payload: {message}}: Constants.SaveAttachment): SagaGenerator<any, any> {
   const {filename, messageID, conversationIDKey} = message
   if (filename && messageID) {
-    const path = downloadFilePath(filename)
+    let path = tmpFile(filename)
```

Comment:
what is this supposed to do logically? if it's a file we download it into their downloads folder but if not we leave it in the tmpfolder?